knowhere/common/ThreadPool.cpp

```diff
@@ -0,0 +1,40 @@
+// Copyright (C) 2019-2020 Zilliz. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance
+// with the License. You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License
+// is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+// or implied. See the License for the specific language governing permissions and limitations under the License
+
+#include "knowhere/common/ThreadPool.h"
+
+#include <memory>
+#include <mutex>
+
+#include "ctpl/ctpl-std.h"
+
+namespace knowhere {
+
+static std::shared_ptr<ctpl::thread_pool> pool_;
+static std::mutex mutex;
+
+void
+ThreadPool::Init(uint32_t num_threads) {
```

Comment:
Since the 2 APIs in this file are very short, suggest to write them to ThreadPool.h